docs/development/testguide.rst

```diff
@@ -227,38 +246,21 @@ Running tests in parallel
 -------------------------
 
 It is possible to speed up astropy's tests using the `pytest-xdist
-<https://pypi.org/project/pytest-xdist>`_ plugin.  This plugin can be
-installed using `pip`_::
-
-    pip install pytest-xdist
+<https://pypi.org/project/pytest-xdist>`_ plugin.
```

Comment:
Again, if installed by default as a dependency, do mention. (If not, that's OK; coverage I do think should be part of it)

Reply:
It isn't included in pytest-astropy so I won't mention this